druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/JDBCExtractionNamespaceCacheFactoryWithLeaderAndFollower.java

```diff
@@ -329,6 +329,10 @@ public Timestamp updateLocalCache(final JDBCExtractionNamespace extractionNamesp
      * @return
      */
     synchronized Producer<String, byte[]> ensureKafkaProducer(Properties kafkaProperties) {
+        kafkaProperties.put(ProducerConfig.ACKS_CONFIG, kafkaProperties.getProperty("acks", "0"));
```

Comment:
why acks is not required?

Reply:
These configs were mirroring Missing Lookups, but all properties will now be passed in the lookup to give the user total control & maintain fewer responsibilities in the lookup.